configs/schema/configJsonSchema.js

```diff
@@ -195,6 +195,9 @@ module.exports = {
           type: ['string', 'null'],
           minLength: 1,
         },
+        debug: {
```

Comment:
I would keep it boolean only and set the default value in the base config. Also, please update the migration.